pkg/rc/replication_controller.go

```diff
@@ -42,7 +43,13 @@ func New(
 	rcStore rcstore.Store,
 	scheduler Scheduler,
 	podApplicator labels.Applicator,
```

Comment:
someone may mix up the order of `podApplicator` and `rcApplicator` and it will suck. Any way to deal with this? I didn't want them to necessarily have to be backed by the same kind of applicator (though in both our tests and our real environments they will be...). So I guess I could just make them a single argument...